src/components/callout/Callout.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import './callout.scss'
+import { computed, useSlots } from 'vue'
+import { className } from '../utlity'
+import '../background/background.scss'
+import Background from "../shared/background";
+import '../size/size.scss'
+import Media from '../shared/media'
+
+const name = 'uids-callout'
+const props = defineProps({
+  size: {
+    type: String,
+    default: 'medium',
+    validator: function (value) {
+      return ['small', 'medium', 'large'].indexOf(value) !== -1;
+    },
+  },
+  alignment: {
+    type: String,
+    default: 'right',
+    validator: function (value) {
+      return ['', 'left', 'right', 'center'].indexOf(value) !== -1;
+    },
+  },
+  ...Background.props,
+  ...Media.props,
```

Comment:
Is there a reason this is being included? I can't see that it is actually being used.

Reply:
Background is being used, but not media.

Reply:
Media props have been removed